libraries/Microsoft.Bot.Builder.LanguageGeneration/LGTemplateLexer.g4

```diff
@@ -0,0 +1,202 @@
+﻿lexer grammar LGTemplateLexer;
+
+@parser::header {#pragma warning disable 3021} // Disable StyleCop warning CS3021 re CLSCompliant attribute in generated files.
+@lexer::header {#pragma warning disable 3021} // Disable StyleCop warning CS3021 re CLSCompliant attribute in generated files.
+
+@lexer::members {
+  bool ignoreWS = true; // usually we ignore whitespace, but inside template, whitespace is significant
```

Comment:
Since our scope is now narrowed down to body only, perhaps there is some space here to optimize our control flow